word_guess.rb

```diff
@@ -0,0 +1,192 @@
+                  /"+"******".colorize(:light_yellow)+"~~~~||"+"*****".colorize(:light_yellow)+"******".colorize(:light_red)+"*********  ******".colorize(:red)+" (.|( ) ✨
+          |       `--____"+"**********".colorize(:light_yellow)+"*********".colorize(:light_red)+"******** ******".colorize(:red)+"  (_Y|_/ "+"** **".colorize(:red)+"
+         / ~~~--_____    ~~~/ "+"******".colorize(:light_yellow)+"**********".colorize(:light_red)+"*********** **".colorize(:red)+"  /_| "+"**      **".colorize(:red)+"
+                     `~~~~~         "+"***".colorize(:light_yellow)+"*********".colorize(:light_red)+"*********".colorize(:red)+"       L    "+" ***".colorize(:red)+"
+                                          **".colorize(:light_yellow)+"**    *****".colorize(:light_red)+"  ***** ****".colorize(:red)+"
+                                            "+"***".colorize(:red)+"
+
+    "
+    elsif @tally == 4
+    puts "
+    <~>                          ___________
+     | |,_____                  (BWAHAHAAHA!)
+           ___`|                 ⁻⁻⁻⁻⁻⁻⁻⁻⁻⁻⁻
+           |"+"('>".colorize(:red)+"|`-__            o
+             ~      ~~~--__   º        "+"**              ***".colorize(:light_red).blink+"
+                   ______  (@|   "+"*".colorize(:light_yellow).blink+"******  *".colorize(:light_red).blink+"***    *******    ******".colorize(:red).blink+"
+                  /"+"******".colorize(:light_yellow).blink+"~~~~||"+"*".colorize(:light_yellow).blink+"***********".colorize(:light_red).blink+"**********************".colorize(:red).blink+"
+          |       `--____"+"*******".colorize(:light_yellow).blink+"******".colorize(:light_red).blink+"*****************************".colorize(:red).blink+"
+         / ~~~--_____    ~~~/ "+"***".colorize(:light_yellow).blink+"******".colorize(:light_red).blink+"******************************".colorize(:red).blink+"
+                     `~~~~~         "+"*".colorize(:light_yellow).blink+"*********".colorize(:light_red).blink+"********************".colorize(:red).blink+"
+                                          "+"****    *********".colorize(:light_red).blink+"*****".colorize(:red).blink+"
+                                            "+"***       ********".colorize(:light_red).blink+"***".colorize(:red).blink+"
+                                                            "+"****".colorize(:light_red).blink+"****".colorize(:red).blink+"
+
+    ".colorize(:red).blink
+    puts "The word was: #{@current_word}".colorize(:magenta)
+    end
+  end
+
+def check_letter
+  #we ask for user input using gets.chomp
+  @tally = 0
+  while @tally < 4
+    puts "Choose a letter at your own risk!"
+    puts "Word: #{@correct_letters.join(" ")}".colorize(:green)
+    letter = gets.chomp
+    if letter.length > 1
+      if letter == @current_word
+        you_win
+        exit
+      end
+    end
+      @already_guessed_letters << letter
+      if @current_word.include? letter
```

Comment:
You _can_ use the `include?` method (and any method) without parenthesis. But for best practice we want to use the parenthesis:
`.include?(letter)`